pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -290,35 +208,44 @@ func libImageName(registry string, lang language, tag string) string {
 	return fmt.Sprintf(imageFormat, registry, lang, tag)
 }
 
+func (w *Webhook) isPodEligible(pod *corev1.Pod) bool {
+	return ShouldInject(pod)
```

Comment:
`isPodEligible` calls `ShouldInject` -- which is the globally available package function now. (prme-3)